lib/Doctrine/ORM/Decorator/EntityManagerDecorator.php

```diff
@@ -53,6 +53,14 @@ public function transactional($func)
         return $this->wrapped->transactional($func);
     }
```

Comment:
This methods probably needs to be deprecated too

Reply:
It implements the interface so should be covered by that 🤔